clade/src/lib.rs

```diff
@@ -3,3 +3,30 @@ pub mod schema {
     pub const FILE_DESCRIPTOR_SET: &[u8] =
         tonic::include_file_descriptor_set!("clade_descriptor");
 }
+
+pub mod sync {
+    use arrow_flight::sql::{Any, ProstMessageExt};
+    use prost::Message;
+
+    tonic::include_proto!("clade.sync");
```

Comment:
Why are we doing this and not just letting the proto be in plain Rust?

Reply:
Fair question—I think a part of the answer is that we're using flight as the actual interface here, and it is protobuf based, so this is too by extension.

On a more practical side we may want to enable clients in different languages using this, and this would facilitate that.